### ISSUE TITLE
Edit Milestone with linked tasks/req/bug/testing

### ISSUE THREAD

**Issue by matariq** [AutomationSolutionz/Framework_0.1#602]
Minor UI change needed to make sure all of the above have same ui display as task

![image](https://cloud.githubusercontent.com/assets/13080536/9424937/2d9a90c8-48cd-11e5-8f93-5f1bce9a3492.png)

![image](https://cloud.githubusercontent.com/assets/13080536/9424938/32fa0486-48cd-11e5-9e26-834f4de0de93.png)

![image](https://cloud.githubusercontent.com/assets/13080536/9424940/38b2e8f2-48cd-11e5-9cbc-f1173a5c1323.png)

**Comment by m3thun** [comment]
@riz-hossain Should I create a label named "UI" which will categorize this kind of ticket?

**Comment by riz-hossain** [comment]
Yes please.Any type of simple ui bugs that are considered cosmetic... alignment issues or button replacement.. can be sorted better with ui tag                                                                                                                                                                                                                                                                                                                                         Sent from my BlackBerry 10 smartphone on the TELUS network.                                                                                                                                                                                                                From: mithunSent: Sunday, August 23, 2015 7:39 AMTo: AutomationSolutionz/Framework_0.1Reply To: AutomationSolutionz/Framework_0.1Cc: Riz HossainSubject: Re: [Framework_0.1] Minor UI change - Edit Milestone with linked tasks/req/bug/testing (#602)@riz-hossain Should I create a label named "UI" which will categorize this kind of ticket?

—Reply to this email directly or view it on GitHub.

**Comment by m3thun** [comment]
@riz-hossain How do you create a label? I couldn't find the option. Is this an admin privilege?

**Comment by minar09** [comment]
https://github.com/AutomationSolutionz/Framework_0.1/labels
